hack/test-integration.sh

```diff
@@ -26,6 +26,7 @@ function cleanup()
 package="${OS_TEST_PACKAGE:-test/integration}"
 tags="${OS_TEST_TAGS:-integration no-docker}"
 
+export GOMAXPROCS="$(grep "processor" -c /proc/cpuinfo 2>/dev/null || sysctl -n hw.logicalcpu 2>/dev/null || 1)"
```

Comment:
Eh... [merge] and we can clean it up later.  I kind of want to make our integration stuff a bit better but not now.

> On Jun 9, 2015, at 10:07 PM, Jordan Liggitt <notifications@github.com> wrote:
> 
> In hack/test-integration.sh:
> 
> > @@ -26,6 +26,7 @@ function cleanup()
> >  package="${OS_TEST_PACKAGE:-test/integration}"
> >  tags="${OS_TEST_TAGS:-integration no-docker}"
> >  
> > +export GOMAXPROCS="$(grep "processor" -c /proc/cpuinfo 2>/dev/null || sysctl -n hw.logicalcpu 2>/dev/null || 1)"
> why not a tiny go program we can capture the output of, e.g. http://play.golang.org/p/E_mkDidnca
> 
> —
> Reply to this email directly or view it on GitHub.
>